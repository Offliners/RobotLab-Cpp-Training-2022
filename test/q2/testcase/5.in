886
sEEM
IL19ES3
daRk
cOMMUNitY
sUrv4v02
interpreTer
CONFIDENTIAL
KNocK
VAcATIoN
LAZY
SUpERSTItIoUS
gOal
weeklY
LauncH
shiny
KInD
haRbOr
surgeRy
fEED
despeRate
CR5YOn
ReCEIVE
cheer
squad
pAyday
harvest
qUiTe
disPleaSe
gRAve
Pay
restore
archive
ORIGiNal
DULl
PHySIcs
SPECULATE
MEANWHILE
PRivATELY
LenS
CoNsCienTIOUs
ModEM
InvestigAte
AwKWArD
belOvEd
DISTANCE
GrACe
ACQUainTANCE
pIlL
SEx
cIRCUMsTAnCE
dIliGenT
MUsCuLAR
MID3L5
ceNsus
14649t03263n
rIDICULOUS
coPy
AcCusATIOn
ImprEss
p0y57CIs8
shade
CABIN
NoVELtY
1AIRY
i5side
pAge
DePREcIaTE
SaViNG
intRiGuing
SuRnAme
INTONAtiOn
H90d9r
COncERN
mustaCHE
TWINKLE
APOLoGiZE
7a999698s
ANalYsis
TrusT
SlEep
deplete
Chill
caLCuLaToR
oVErT4RN
danGerous
1881IT636
shALLoW
SERviCE
CoNSErvatION
fortune
legally
AbSORB
COnSultaTion
lEaKaGE
grASS
remember
Conversation
PLOW
pER9E84a6E
ResENT
occasional
br4sh
PHONe
iNdUstrious
68Moral
OvErsEAs
sticKy
TemPEST
silenTly
iNhabitant
REGaRDING
KEEP
s06t6
kICk
sUMmaRY
OFfer
NICe
alley
pOSTEr
cashier
glasses
BUNCH
TRICkY
ExtRACT
INVADE
adverTIse
bRUSh
Pour
appalling
FuturE
SuRROuNDINGs
drY
ConverTIble
EvEnIng
periodic
pREsence
bookcaSE
TeenAger
jOy
CArEss
CommENd
cAPTIve
sITuAteD
tEnt
5O6914
TRaNSFeR
NaTIONAL
parade
REpay
costly
OriENTal
sTREEtcAR
ATtaCk
MEdAl
Cheerful
BaRgAIN
GRind
ROAR
ConTEMPT
ACE
pLUG
CuriOUs
HUMIDITy
coNsCiouS
HoW
VicTORY
n
tRAnsCenD
drop
giFT
530P12
laNDLorD
performance
THReat
7ERV8
JewEl
MEDIUM
burst
baM
PRESIDE
swEET
CLIMATE
patH
Armchair
BeT
COLOR-bLIND
bElie49
gROWth
king
9U350690
OUTRAgE
ImPrAcTical
decReASe
STArtle
CAMPUS
POSSEsS
infErIoR
VanILLA
exPlOSiVe
defeCt
FARAwAy
eNGInE
DISAppoinT
brutaLitY
coNQUER
biOgraPhy
viDeo
advANCE
campaign
RELAX
DiaMond
BOunCe
RAINBow
sy3t9m
HaRm
aDditionAl
R
MENTiON
CoMMERCE
PLAguE
hallway
ChimPanzeE
tUesday
coRRESPOnDEnt
CONVOY
RaT
S
widE
reBEl
budGet
SMelL
STEAM
trAnsPlANT
OBJECTION
impresSIoN
rOBOT
whereabouts
CuPboard
gROvE
ANeCDOTe
PSYCHIATRY
noise
aerospAce
quiT
oWl
VeT
cASUALTy
BalaNce
enable
DONE
COASTLINE
umbRella
artifiCial
iNvesTiGaTiOn
VicTImiZe
1ooN
Rub
CHeCKUP
cOke
discard
sEPtemBer
ShOULDeR
MiSCELLAneoUs
CApaBLE
larGE
CLASSIc
aprOn
ExCeLlenT
sCHOLAR
Lotus
SElect
amaZe
bEwAre
reveaL
o5170Or
ATtITUDE
cRaCKDOwN
surGeon
EXact
IronY
VOCATION
GRa76
mASTER
STOol
quest
SIMPlE
miraculouSly
grAIn
seldom
sleePy
T1P4ca5
PUb
goldeN
exPerimenT
JUST
ROUnd
bleak
envioUs
lOyaltY
begIN
pop
COlLiSiOn
sNEAK
UnDer
2esit11t08
vulnErable
HARASsMENT
bAmBOO
MEN
396oi88m49t
partIalLy
TOTALLY
approve
ANy
hook
BLOCK
AMuSe
read
typhoon
PEDAL
MEan
GENE
de34e28t8
pENaLtY
WeEd
OX
literature
WHENEVER
BARRIEr
AMbitiOn
monitor
SCRuTINy
CaUSe
26ON5
slug
contend
eaRThquAKE
peneTraTinG
FESTIvaL
silk
BEeF
miLiTANt
ViRTUOUS
rhyThmiC
REGIonAL
BUlLY
ITALIAN
scare
hyPocRiSY
LATITUDE
PrecaUt1On
peR3ENT
25e6
hUmid
092T7m8C
WHATEVER
DArling
PREVENTIVE
pULl
80BE037
BELL
EARNingS
reAliSTIC
coMpartment
EAGer
CoMPAtIBLY
AFtER
0OND
misSing
rUIN
WeSTERn
scRaP
rEPLy
wEaRy
PreSIdENTiaL
distINCtiON
EnvelOpe
6CLI616
LeSbian
BeneFit
oCcupaTIOn
RaZoR
iceberg
PROfIT
aDmiRAL
pERfORmER
sUpeRFICIAL
CuRiOSiTY
POSiTIvE
mOUrN
LeADERSHIP
repUtATIoN
SEVENTY
HOWL
MarK
joIN
RedUce
ELEMENTAl
RAG
DIALOGUe
hARS84Y
HOsPItAL
dISAdVANTAGE
WhatsoeVer
CHEmicAL
adDrESS
gi6ted
MEADOw
tow
dRINK
PREMIER
PAct
dEaL
exPErienCE
N9Mi3eE
nA3urA4
cOWORKER
historian
palm
sEVERAl
brochure
881I3628
FACULTY
WARN
doG
THReAd
2AsP
PANACeA
GazE
COnTrACtOr
STrATEGIC
MAstERpIECE
susPicIoUs
SOCIALLY
CASh
seNtimental
BUNdLe
explanation
hystericAl
HONORABlE
BOoT
ForMulA
noWADaYs
polIce
piNEaPplE
CARdinaL
knife
COWardice
8RO5INC6AL
ACCIdENT
heaDquArteRs
UTmOSt
REbEl
Cycle
8c8te
COUPoN
Dearly
DIvORCE
RoLe
eXHaUst
Air-Tight
STAndArd
5OS88014
PHILoSOphIcaL
CUrRENT
dePrEsSION
eLSEWhErE
faIluRE
ADMINISTRATIVE
CaRrOT
CONFER
aLoNgsIdE
SHADY
ANyWAy
PROVIde
sENse
innovation
appetite
StIMUlAtiON
AiDS
O3v36UsLY
THUNdErSTORm
MEDIEVAL
shape
BEHALF
31AiR3606N
GirAffe
OPTImiSTIC
Blunder
moment
COAL
ClOsE
SuNLigHT
precious
OMeN
WaTerMELOn
ENCOUNTER
socceR
REFORM
Clip
052t0r6
actIoN
4v5il7b96ity
sCReAM
47I6H33R
circus
INDUlGE
AmbIGuItY
STEREOTYPE
LoVER
DRESS
wealtH
indifferEnce
coMmuNiCATiON
sTAGE
etHic
0a9893
cucUmber
when
pIck-uP
106Ent
aLteRnAtive
EDIBLE
cOUNTy
8n9t
week
elas9I9
pa5nS14p
BEwildeR
ExCeEDINGLY
perMissiOn
63921a57
hINt
Represent
dismiss
shrimp
4r1n7P6436
conservative
choRus
IsolatiOn
EiGHt
iNdeciSion
SlOw
InExpenSIV4
baG
TiME
theOloGIaN
deNoTe
USaGE
cOnnect
BULL
elBOw
rIVAl
appOinTMeNt
eNDOrse
DIsTRICT
LoOSE
aBSeNCe
ReacTionaRy
cOmmutE
mUsEUM
baCheLOr
rAdiAte
mooN
SEnSItiVE
4M7273E
gasp
AboARD
Weak
Economy
CONSidERATIoN
u1A0aRe
s36985Ht
Awkwardly
SyNthesiS
CLAIM
UNDEreStIMATE
6u5il
VIne
EGO
SOFA
OBJECT
sIBliNg
cOOkeR
PIEce
SiGnIFICAnT
mEasURe
bReAkfast
CoNCrETE
RECEIPT
FU29
STrAIN
bREaThE
MAGICAL
JUstiCe
aGRi1U3T2R5
uNLIkE
ULCER
swiTcH
DIsPoSAl
infLueNZA
Distort
SUpE4Io1
forty
pROcEss
CUMUlATIVE
gaRDEn
recovery
allieD
LISTenER
EVOLuTION
QuIZ
95on6U3C8
DEVOTE
plea
CReDIT
thOse
cOURaGE
sigh
fORbid
tOILet
TOOTHPASTE
FAREWELL
aUGUST
reside
gOOsE
kiSS
prOvE
coNSuME
consumption
backward
multipLe
VARIETY
c
GRACEFUL
Chop
dust
CoNCeNTRATE
GLANCE
track
strEak
RUBBER
supermarket
SIGHTSEEing
foRmAl
spadE
InEViTABle
SettlE
DEPORT
HysterIcal
lAST
continue
coNvEntIoN
LaBORATORy/lAB
secoNDAry
rule
OFficIaL
ETEMITY
DraPe
hATReD
HaIL
strAteGy
counteR
mansion
KIL4
surround
LeCTurE
SoveReIGN
DORmITORy/DORm
dRead
COURTEsy
encyclopedia
buy
lobster
expLAiN
CoEXIST
CONDITION
CoRPoration
CALCULATiOn
melancholy
RAIlWAY
AGREE
VERSE
EDUCATE
913M041
sPeLLINg
deceptive
ThemE
85il82
dYnamiTE
FoG
basemenT
SuItcAse
misErY
MOoD
telegram
DOoM
eneMy
UNFORGETTABLE
5249AL
MALICiOUS
caNCel
commUtEr
PaPAYA
4a7318
bLoom
mop
BEHoLD
ACCELERATION
embassy
roYaLTy
DISTInCT
GROAN
MArcH
ANTIBIOtIc
CrAdle
uPenD
ForEiGnEr
FAIL
EXHIBITION
ULTRAVioleT
INsECURE
771402p
LONG
LiKElIhOod
lattEr
dismantle
GROW
restroOM
mOtIVAtiOn
LAKe
IdENTITy
DISpOSAble
DiVInE
suCk
EcOnoMIC
stoNe
TEsTIFY
cuBIc
diSAGREe
T8n15T
FICTIOn
sUit
ADvENtuRE
I227NT78E
divErsity
Camp
irriTAte
FRoWN
REIn
BRACE
i
d5sT2R5
feet
sHAme
CELEBRATE
INVEST
perPLeX
lOcatiOn
AngulAR
juiCe
gRAte
P1ACH
comB
huNDReD
chROnIc
MOTEl
caReer
unAniMOUSlY
chase
gUIltY
Ma60EmatIC3/6a3h
BOTHER
pITch
remAiN
LID
tEnd
glItTEr
VERdiCT
temperatuRe
SURvEY
DOUBLE
CONVERT
daily
terminal
HONEsT
pine
SoLdier
wAIST
DeScendaNT
tiE
VivId
AcCePTabLE
hole
Thief
02r36c69
PARtNER
MOSQUITO
InC14590
BiNGo
GHETTO
iNvESTOr
marble
smooth
BOrING
5NcOME
dIsCO
Crisis
scaLe
photographer
pRob5b8e
cLUTCh
posture
DuTY
coNfeSsion
bEard
N
desseRt
rEPRESENTATION
New
FoggY
CLIFF
nONsENSe
sILEnCE
2B6TR06TI7n
DIALECT
STURDY
CONCESSION
CAVAlRY
casual
wHO
LA2
forecast
bREathTAkiNG
AlLot
sIllY
DRUNK
UnpreCeDEnTEd
paralyze
mAil
resignaTioN
soon
SUIcIdE
bOOm
CiRCLE
goalkeeper
t8To6
glaSs
leaRNeD
inItIAlLy
Idol
odDs
PAsTURE
eighTy
bOwLIng
DOubtlEss
consolidate
HoUSInG
SUpERFLUOUS
RESPONse
mINUS
U8LoAD
CENT
DEpiCt
never
CaUTIon
cOnnEctiOn
rebeLlion
3d9
MAYoR
auTHoritY
SODA
fIST
SCARED
5I7A5
LiST
dISguSt
applIcatiOn
uNCLe
FAtAl
PLOT
tELEGRAPh
capitalist
whom
daZ1LE
ORgAniZe
