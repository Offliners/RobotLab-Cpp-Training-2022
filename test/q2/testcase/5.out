v. 似乎
Unknown
adj./n. 黑暗的
n. 社區；共同體
Unknown
n. 口譯者
adj. 機密的
n./v. 敲；打
n. 假期
adj. 懶散的
adj. 迷信的
n. 目標
adj./adv. 一週一次的
v./n. 發射；投擲
adj. 閃爍的
adj. 親切的
n. 海灣；港灣
n. 外科(手術)
v./n. 餵食
adj. 絕望的
Unknown
v. 收到
n./v. 歡呼；激勵
n. 爭吵
n. 發薪日
n./v. 收穫
adv. 完全；非常
v. 使不快
adj. 重大的；嚴肅的
n/v. 付錢
v. 歸還
n. 檔案
adj./n. 最初的；原著
adj. 單調的；遲鈍的
n. 物理學
v. 推測；投機
adv. 其問；同時
adv. 私下地
n. 鏡片；眼球的水晶體
adj. 盡責的
adj. 現代的
v. 調查；研究
adj. 笨拙的；棘手的
n. 所愛的人
n. 距離
n./v. 優雅
n. 熟人
n. 藥丸
n. 性(別)
n. 情況；詳情
adj. 勤奮的
adj. 肌肉的
Unknown
n. 調查
Unknown
adj. 可笑的
n./v. 副本；拷貝
n. 控訴
v. 使銘記
Unknown
n. 陰蔭
n. 客艙
n. 新鮮；新奇
Unknown
Unknown
n. 頁
v. 跌價
n. 存款
adj.  有趣的；有魅力的
n. 姓
n. 語調
Unknown
n./v. 關心；涉及
n. 鬚
n./v. 閃爍
v. 道歉
Unknown
n. 分析
n./v. 信任
n./v. 睡眠
v. 使枯竭
v./adj. (使)變冷；寒冷的
n. 計算機
Unknown
adj. 危險的
Unknown
adj. 淺的
n. 服侍
n. 保護
n. 財產；好運
adv. 法律上
v. 吸收；理解
n. 協商會
n. 洩漏
n. 青草；草地
v. 記得
n. 會話；談吐
v. 耕(地)
Unknown
v. 不滿
adj. 偶爾的
Unknown
n./v. 電話
adj. 勤勉的
Unknown
adj./ad v. 來自海外的
adj. 黏的
n. 暴風(雨、雪)
adv. 寂靜地
n. 居民
prep. 關於…
v. 保存
Unknown
v. 踢；反衝
n. 摘要
n/v. 提議；提供
adj. 好的
n. 小巷子；小徑
n. 海報
n. 出納員
n. 眼鏡
n. 一束；一群
adj. 狡猾的
v. 拔出；摘錄
v. 侵略；侵犯
v. (n)為…做廣告
n./v. 刷子；刷
v.灌 v.注 v. 倒
adj. 令人恐懼的
n./adj. 未來；前途
n. 環境
adj./v. 乾燥的；使變乾
adj. 可轉換的
n. 晚上
adj. 定期的
n. 出席
n. 書架；書櫥
n. 青少年
n. 歡樂
v. 愛撫
v. 稱讚
adj. 被俘虜的
adj. 坐落在…的
n. 帳篷
Unknown
n./v. 調動
adj. 國家的
n. 遊行
v. 償還
adj. 貴重的；代價高的
n./adj. 東方人；東方的
n. (市內)有軌電車
v./n. 攻擊
n. 獎章；紀念章
adj. 興高采烈的
n./v. 買賣；便宜貨
v. 磨；碾；壓
n./v. 怒吼
v. 輕視
n. 一點(撲克牌)；能手
n. 塞子
adj. 好奇的
n. 溼氣
adj. 神智清醒的
adv. 如何；多少
n. 勝利
Unknown
v. 超越
n./v. (一)滴；下降
n. 禮品；天賦
Unknown
n. 地主
n. 成就；表演
n. 威脅
Unknown
n. 珠寶
n. 中間；媒介
v./n. 爆發；爆炸
n. 穀倉；牲口棚
v. 主持
adj. 甜的
n. 氣候；風氣
n. 小徑
n. 扶手椅
n./v. 打賭
adj. 色盲的
Unknown
n. 生長；發育
n. 國王
Unknown
n. 暴行
adj. 不切實際的
v./n. 減少
v. 使大吃一驚
n. (大學的) 校園
v. 擁有
adj. 低於…的
n. 香草
adj./n. 爆炸性的
n. 缺點
adj. 遠方的
n. 發動機；引擎
v. (n)使失望
n. 野蠻
v. 戰勝
n. 傳記
n. 錄影帶
n./v. 推進；將…提前
n./v. 競選活動；戰役
v. 放鬆
n. 鑽石
n./v. 跳，躍；使反彈
n. 彩虹
Unknown
n./v. 損害；傷害
adj. 附加的
Unknown
n./v. 提及
n. 商業；交易
n./v. 瘟疫；折磨
n. 玄關；走廊
n. 黑猩猩
n.星期 v. 
n./adj. 通訊(記)者；一致的
n. 護送(車隊)
n. 鼠
Unknown
adj. 寬的
n. 反抗者
n./v. 預算
n./v. 氣味；聞
n./v. 蒸汽
v./n. 移植
n. 反對
n. 印象
n. 機器人
adv. 在哪兒
n. 碗櫃
n. 樹叢
n. 軼事
n. 精神病學
n. 噪音
n. 航空太空(包括大氣層內外)
v. 停止
n. 貓頭鷹
n./v. 獸醫；審查
n. 傷亡者
n./v. 平衡
v. 使能夠
adj. 完成的
n. 海岸線
n. 雨傘
adj. 人造的
n. 調查；研究
v. 使犧牲
Unknown
v. 摩擦
n. 檢查
n. 可口可樂
v. 摒棄
n. 九月
n./v. 肩膀；肩負
adj. 各式各樣的
adj. 有能力的
adj. 大的
adj./n. 經典的；名著
n. 圍裙；工作裙
adj. 傑出的；卓越的
n. 學者
n. 蓮；忘憂樹
v. 挑選
v. (n)使驚愕
v. 當心
v. 揭露
Unknown
n. 態度
n. 取締
n. 外科醫師
adj. 精確的
n. 反語
n. 職業
Unknown
n. 主人；雇主
n. 凳子
n. 探索
adj. 簡單的
adv. 神奇地
n. 穀粒；穀物
adv. 很少；不常
adj. 想睡的
Unknown
n. 小酒館
adj. 金色的；貴重的
n./v. 實驗
adj. 公平的
adj./n. 圓的
adj. 陰冷(鬱)的
adj. 妒嫉的
n. 忠誠
v. 開始
n./v. 砰然(聲)
n. 衝突
v. 鬼鬼祟祟地走
prep./adv. 在…下面
Unknown
adj. 脆弱的
n. 煩惱
n. 竹子
n.  man的複數
Unknown
adv. 部分地
adj. 完全地
v. 同意；認可
adj./adv./pron. 任何的；若干
n. 勾子；陷阱
n./v. 塊；障礙物；堵住
v.(n. )給～提供娛樂
v. 讀
n. 颱風
n. 踏板
adj. 吝嗇的
n. 基因；遺傳因子
Unknown
n. 處罰
n. 雜草
n. 公牛
n. 文學
adv./conj. 無論什麼時候；每當
n. 障礙物；棚欄
n. 抱負；志向
n./v. 監視器；監控
n. 細查
n./v. 導致；原因
Unknown
n. 子彈；蛞蝓
v. 競爭；鬥爭
n. 地震
adj. 敏銳的
n. 節日；歡樂
n. 絲
n. 牛肉
adj. 好戰的
adj. 品德高尚的
adj. 韻律的
adj. 地區的
n. 恃強凌弱者
n. 義大利文
v. 恐嚇
n. 虛偽
n. 緯度
Unknown
Unknown
Unknown
adj. 潮溼的
Unknown
adj./advJpron./conj.  任何的；無論什麼
adj./n. 親愛的
adj. 預防的
v. 拖；拉
Unknown
n. 鐘(聲)；鈴(聲)
n. 收入；工資
adj. 理論的
n. 分隔間
adj. 熱切的
adv. 和諧地
prep./adv./adj. 在…之後；以後的
Unknown
adj. 行蹤不明的
n./v. 毀滅
adj./n. 西方世界
n. 碎片
n./v. 答覆
adj. 筋疲力竭的
adj. 總統(校長)的
n. 區別
n. 信封
Unknown
n. 女同性戀者
n./v. 利益；有益於
n. 職業；占用
n. 舌」鬍刀
n. 冰山
n./v. 利益；有益於
n. 海軍將官
n. 執行者；表演者
adj. 表面(皮)的
n. 好奇心
adj. 確信的；積極的
v. 悲悼
n. 指揮；統率力
n. 名聲
adj./n. 七十
v. 號叫
n./v. 痕跡；記號
v. 參加
v. 減少
adj. 自然力的
n. 破布
n. 對話
Unknown
n. 醫院
n. 不利之條件
adj. 任何的
adj./n. 化學的
n./v. 地址；致詞
Unknown
n. 草地
v./n. 拖
n./ v. 飲料；喝
n. 首相
n. 協定；公約
n ./v. 交易；處理
n./v. 經驗
Unknown
Unknown
n. 同事
n. 歷史學家
n. 棕櫚；椰子
adj./n. 幾個的；幾個
n. 小冊子
Unknown
n. 官能；技能
v. 警告
n. 狗
n./v. 線；穿線於
Unknown
n. 萬靈藥
v. 凝視；注視
n. 立約人
adj. 戰略(性)的
n. 傑作；名著
adj. 猜疑的
adj. 善於交際地
n./v. 現金
adj. 感情上的
n. 捆；包裹
n. 解釋
adj. 患歇斯底里症的
adj. 榮譽的
n. (長筒)靴
n. 配方；客套話；準則
adj. 現在；今日
n. 警察
n. 鳳梨
n. 紅衣主教
n. 刀
n. 膽小
Unknown
n. 事故
n. 總部
adj. 極度的
n. 反抗者
n./v. 週期；循環
Unknown
n. 優待券
adv. 深切地
n./v. 離婚
n. 角色
v. 汲乾；使精疲力盡
adj. 密封的
adj./n. 標準
Unknown
adj. 哲學的
adj./n. 目前的；潮流
n. 沮喪
adv. 在別處
n. 失敗
adj. 行政的
n. 胡蘿蔔
v. 商議；授予
prep./adv. 靠著…旁邊
adj. 遮蔭的
adv. 無論如何
v. 提供
n./v. 感官；覺得
n. 革新；創新
n. 胃口；食慾
n. 刺激
n. 愛滋病
Unknown
n. 雷雨
adj. 中古的；中世紀的
n./v. 形狀；做成
n. 為了…的利益；代表
Unknown
n. 長頸鹿
adj. 樂觀的；有信心的
n. 愚蠢的錯誤
n. 片刻
n. 煤礦；煤塊
adj./adv. 接近的
n. 陽光
adj. 珍貴的
n. 微兆；預兆
n. 西瓜
v./n. 遭遇；衝突
n. 足球
v. 改革
v. 修剪；夾住
Unknown
n. 行動；作用
Unknown
n./v. 尖叫聲；尖叫
Unknown
n. 馬戲團
v. 使滿足
n. 意義含糊
n. 老套
n. 戀人
n./v. 女裝；打扮
n. 財富
n. 不關心
n. 溝通；訊息
n./v. 舞臺；上演
n. 倫理
Unknown
n. 黃瓜
pron./adv./conj.  那時；什麼時候
n. 擴音器
Unknown
adj. 替代的；二擇一的
adj. 可食的；食用的
n. 郡；縣
Unknown
n. 週
Unknown
Unknown
v. 使混亂
adv. 極其
n. 准許
Unknown
n./v. 暗示
v. 描寫
v. 解散
n. 小蝦；矮子
Unknown
adj. 保守的
n. 合唱隊
n. 隔離
adj. /n八
n. 無決斷力
adj. 緩慢的
Unknown
n. 袋子
n./v. 時間
n. 神學家
v. 意味著
n. 使用方法
v. 連結
n. (未閹割的) 公牛
n. 肘部；扶手
n. 對手
n. 約定；指派
v. 贊同
n. 區域
adj. 鬆散的
n. 缺席；缺少
adj. 保守的；反動的
v. 通勤
n. 博物館
n. 單身漢；學士
v. 發射
n. 月亮
adj. 敏感的
Unknown
v. 喘息
prep./adv. 在(船、飛機、車)上
adj. 虛弱的
n. 節約；經濟
n. 考慮；動機
Unknown
Unknown
adv. 笨拙地
n. 綜合(體)；合成
n./v. 要求；聲稱
v. 低估
Unknown
n. 葡萄(藤)
n. 自我
n. 沙發
n. 物體；目標
n. 手足
n. 炊具
n./v. 破片；拼合
adj. 重要的
v./n. 測量
n./v. 早餐
adj./n. 具體的；混凝土
n. 收到；收據
Unknown
v. 扭傷
v. 呼吸
adj. 魔法的
n. 公平；正義
Unknown
prep. 不像
n. 潰瘍
n./ v. 開關；轉變
n. 處理；配置
n. 流行性感冒
adj. 歪曲
Unknown
adj./n. 四十
n./v. 過程；加工
adj. 累積的
n./v. 花園；從事園藝
n. 復原
adj. 同盟的
n. 聽眾
n. 演化(論)
n. 問答比賽
Unknown
v. 將…奉獻給
n. 懇求
n./v. 信用；賒帳
adj./pron. 那些
n. 勇氣
n./v. 嘆息
v. 禁止；阻止
n. 洗手間；廁所
n. 牙膏
n. 告別；告別詞
n. 八月
v. 居住
n. 鵝；傻瓜
n./v. 吻
v. 證明
v. 消耗；花費
n. 消費
adv. 反方向地
adj. 多重的
n. 多樣化
Unknown
adj. 優美的
v./n. 劈；肋排肉
n./v. 灰塵；打掃
v. 專心
v. 一瞥；掃視
n./v. 軌道；追蹤
n./v. 條紋；快速移動
n. 橡膠
n. 超級市場
n. 觀光
adj. 正式的
n. 鏟
adj. 無可避免的
v. (n〕決定
v. 驅逐出境
adj. 患歇斯底里症的
adj./adv./n./pron.  最後的；最近的
v. 持續
n. 慣例；會議
n. 實驗室
adj. 次要的
n./v. 規則
adj./n. 官方的；官員
n. 永恆
v. 覆蓋
n. 憎恨；敵意
v. 歡迎
n. 策略
n. 櫃檯；籌碼
n. 大廈；官邸
Unknown
v. 圍繞
n./v. 授課
n. 元首
n. 學生宿舍
n./v. 害怕
n. 有禮的舉止
n. 百科全書
v. 買
n. 龍蝦
v. 解釋
v. 同時共存
n./v. 情況；決定
n. 公司
n. 計算
n./adj. 悲哀；憂鬱
n. 鐵路
v. (n)同意
n. 詩節
v. 教育；教導
Unknown
n. 拼寫能力
adj. 虛偽的
n. 主題
Unknown
n. 甘油炸藥
n./v. 霧氣；以霧籠罩
n. 地下室
n. 手提箱
n. 不幸；悲慘
n. 情緒
n. 電報
n./v. 厄運；注定
n. 敵人
adj. 難忘的
Unknown
adj. 存心不良的
v. 取消
n. 通勤者
n. 木瓜
Unknown
n./v. 花；開花
n./v. 拖把
v. 見到
n. 加速
n. 大使館
n. 皇室成員
adj. 有區別的
n./v. 呻吟
n. 三月
n. 抗生素
n./v. 搖籃；撫育
v. 把…倒置
n. 外國人
v./n. 失敗；不及格
n. 展示會
adj. 紫外(線)的
adj. 不安全的
Unknown
v. 渴望
n. 可能性
adj./n. 後面的；後半的
v. 拆卸
v. 發育；種植
n. 廁所
n. 動機
n. 湖
n. 身分；同一性
adj. 可丟棄的
adj. 神的；神聖的
v. 吸；吮
adj. 經濟上的
n./v. 石頭
v. 作證
adj. 立方的
v. (n)不同意
Unknown
n. 小說；虛構的事
n. 一套衣服
n. 冒險
Unknown
n. 多樣性
n./v. 營地；露營
v. 激怒
v./n. 皺眉；表示不滿
n. 韁繩；統治
n. 支撐物
Unknown
Unknown
n. [pl. ]腳
n./v. 恥辱
v. 慶祝
v.(n. )投資；投入
v. 使困惑
n. 選定路線
adj. 有尖角的
n. 果汁
n./v. 壁爐；磨碎
Unknown
n./v. 梳子；梳頭
adj./n. 一百
adj. 慢性的
n. 汽車旅館
n. 職業
adv. 全體一致地
v./n. 追逐
adj. 有罪的；內疚的
Unknown
v./n. 打擾；煩惱
n. 瀝青
v. 殘留
n. 蓋子；眼瞼
v. 有…的傾向　
v. 閃爍
n. 判決
n. 溫度
n./v. 調查；審視
adj./adv./n./v. 兩倍的；雙重地
v. 轉換
adj./adv. 每日的
n. 終點；總站
adj. 誠實的
n. 松樹
n. 軍人
n. 腰
n. 後代
n. 領帶
adj. 生動的
adj. 可以接受的
n. 孔；漏洞
n. 小偷
Unknown
n. 伙伴
n. 蚊
Unknown
n. 賓果遊戲
n. 城市中猶太人的保留區
n. 投資者
n. 大理石
adj. 光滑的
adj. 令人厭煩的
Unknown
n. 迪斯可舞廳/音樂
n. 危機
n. 標準
n. 攝影者
Unknown
n. 離合器；緊握
n. 姿勢
n. 義務；稅
n. 承認
n. (下巴上的)鬍鬚
Unknown
n. 甜點；餐後點心
n. 象徵
adj. 新的
adj. 多霧的
n. 懸崖
n. 胡說
n./v. 寂靜
Unknown
n. 方言
adj. 強健的
n. 讓步
n. 騎兵(隊)
adj. 偶然的；碰巧的
pron. 誰
Unknown
n./v. 預測；預言
adj. 使人興奮的
v.  分配
adj. 愚蠢的
adj./n. 喝酒醉的
adj. 前所未有的
v. 使麻痺；使癱瘓
n./v. 郵件；郵寄
n. 辭職；辭呈
adv. 很快
n. 自殺
n. 隆隆聲；景氣
n./v. 循環；畫圓圈
n. 守門員
Unknown
n. 玻璃；玻璃杯
adj. 有學問的
adv. 在起初
n. 偶像
n. 可能的機會
n. 牧地；草原
adj./n. 八十
n. 保齡球
adv. 無疑地
v. 鞏固
n. 住房建築
adj. 過多的
n. 回答
prep./adj./n. 負的
Unknown
n. 一分錢
v. 描繪
adv. 永不
n. 謹慎
n. 連接；關聯
n. 造反
Unknown
n. 市長；鎮長
n. 權力；當局
n. 蘇打水
n. 拳頭；支配
adj. 害怕的
Unknown
n./v. 表；把…編列成表
n./v. 嫌惡；使厭惡
n. 應用；申請書
n. 伯父
adj. 致命的
n./v. 情節；圖謀
n. 電報
n. 資本家
pron. 誰
Unknown
v. 組織
